gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GrpcChannelUtilsTest.java

```diff
@@ -0,0 +1,29 @@
+package com.google.cloud.hadoop.gcsio;
+
+import static com.google.cloud.hadoop.gcsio.GrpcChannelUtils.V2_BUCKET_NAME_PREFIX;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import org.junit.Test;
+
+public class GrpcChannelUtilsTest {
```

Comment:
Add `@RunWith(JUnit4.class)` annotation for consistency with rest off the tests.

Reply:
Done